operator/v1/types_csi_cluster_driver.go

```diff
@@ -168,6 +168,63 @@ type AWSCSIDriverConfigSpec struct {
 	// +kubebuilder:validation:Pattern:=`^arn:(aws|aws-cn|aws-us-gov|aws-iso|aws-iso-b|aws-iso-e|aws-iso-f):kms:[a-z0-9-]+:[0-9]{12}:(key|alias)\/.*$`
 	// +optional
 	KMSKeyARN string `json:"kmsKeyARN,omitempty"`
+
+	// efsVolumeMetrics sets the configuration for collecting metrics from EFS volumes used by the EFS CSI Driver.
+	// +openshift:enable:FeatureGate=AWSEFSDriverVolumeMetrics
+	EFSVolumeMetrics AWSEFSVolumeMetrics `json:"efsVolumeMetrics"`
+}
+
+// AWSEFSVolumeMetricsState defines the modes for collecting volume metrics in the AWS EFS CSI Driver.
+// This can either enable recursive collection of volume metrics or disable metric collection entirely.
+// +openshift:enable:FeatureGate=AWSEFSDriverVolumeMetrics
```

Comment:
You don't need this here, only where the fields are brought into the API, at the highest level, so, only on L173 at present